pytorch_module_path: "/home/dyros/Workspace/GIST/assembly_part_segmentation/furniture_part_segmentation"
config_file: "rgb.json"
weight_path: "/home/dyros/catkin_ws/src/assembly_part_recognition/pretrained_weights/furniture_part/0814_snu_furniture_part.tar"
rgb: "/azure1/rgb/image_raw"
width: 1280
height: 960
debug: true
is_thresh: 0.9
roi: [250, 800, 330, 830]
